### ISSUE TITLE
Evaluation fail!

### ISSUE THREAD

**Issue by ThaiLe189** [Bartzi/stn-ocr#31]
Hello Bartzi 
I try to run evaluation SVHN and i get this error:
![Screenshot from 2020-08-17 13-49-29](https://user-images.githubusercontent.com/58130773/90377274-fc91ee00-e0a1-11ea-8756-d2f165cace5e.png)

the command: 
python eval_svhn_model.py '/home/hthai/stn-ocr/datasets/svhn/original_svhn/models/model' 0040 '/home/hthai/stn-ocr/datasets/svhn/evaluation/test.csv' '/home/hthai/stn-ocr/datasets/svhn/svhn_char_map.json'

How can i fix it ?
Thank you !

**Comment by Bartzi** [comment]
Sorry that it took me so long to answer!

That is definitely difficult to say.
What version of MXNet are you using? Did you make any changes in the code?

**Comment by ThaiLe189** [comment]
Yeah i using another version of mxnet because when i compile mxnet with tag v0.9.3, i get error

The command i was ran:
git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet1
git checkout v0.9.3
#edit config file in mxnet

![Screenshot from 2020-08-27 13-44-13](https://user-images.githubusercontent.com/58130773/91403316-85abe080-e86b-11ea-82a2-762151dc8a77.png)

git submodule update --init --recursive
make -j4 \USE_OPENCV=1 \USE_CUDA=1 \USE_CUDA_PATH=/usr/local/cuda \USE_CUDNN=1 \USE_BLAS=openblas

![Screenshot from 2020-08-27 13-50-24](https://user-images.githubusercontent.com/58130773/91406191-55187680-e86c-11ea-8b2f-d99c22c958f5.png)


in image is mxnet1 because i don't want to remove version of mxnet i was compiled

**Comment by ThaiLe189** [comment]
I realize that compile mxnet with cuda give me that error, i have try compile mxnet with command:
make -j4 \USE_OPENCV=1 \USE_BLAS=openblas
and it work but i can't use my gpu for train
I using Cuda version 10.1 and cudnn 7.6.5

Update i try use Cuda 9.0 but have same error

**Comment by Bartzi** [comment]
Hmm, I think I can not help with that then, since I only tested the code with the version of MXNet showed in the README.
I suggest that you have a look at my other implementation with Chainer from the paper [SEE](https://github.com/Bartzi/see). This should work better on newer systems, since chainer is a nicer framework.

**Comment by Bartzi** [comment]
Oh and regarding your last screenshot: What is the actual error? I can only see warnings and that there is an error, while compiling `cuddn_convolution`

**Comment by ThaiLe189** [comment]
I compile mxnet v0.9.3 with command: make -j4 \USE_OPENCV=1 \USE_CUDA=1 \USE_CUDA_PATH=/usr/local/cuda \USE_CUDNN=1 \USE_BLAS=openblas 
and i get that error, i don't know actual error
So let me check Chainer first 
Thank you very much.
